ui/apps/platform/src/Containers/Dashboard/Widgets/ComplianceLevelsByStandard.tsx

```diff
@@ -72,27 +61,29 @@ function processData(
     if (!data) {
         return undefined;
     }
-    const { complianceStandards } = data;
-    const modifiedData = data.controls.results.map((result) => {
+    const { complianceStandards, controls } = data;
+
+    const modifiedData: ComplianceLevelByStandard[] = [];
+    controls.results.forEach((result) => {
         const aggregationName = result?.aggregationKeys[0]?.id;
         const standard = complianceStandards.find((cs) => cs.id === aggregationName);
-        const { numPassing, numFailing } = result;
-        const standardQueryValue =
-            standardLabels[standard?.id] || aggregationName || 'Unrecognized standard';
-        const query = getUrlQueryStringForSearchFilter({
-            ...searchFilter,
-            Cluster: searchFilter.Cluster || '*',
-            standard: standardQueryValue,
-        });
-        const link = `${complianceBasePath}/${
-            urlEntityListTypes[standardEntityTypes.CONTROL]
-        }?${query}`;
-        const modifiedResult = {
-            name: standard?.name || aggregationName || 'Unrecognized standard',
-            passing: Math.round((numPassing / (numFailing + numPassing)) * 100) || 0,
-            link,
-        };
-        return modifiedResult;
+
+        if (standard) {
+            const { numPassing, numFailing } = result;
+            const standardQueryValue =
+                standardLabels[standard.id] || aggregationName || 'Unrecognized standard';
+            const query = getUrlQueryStringForSearchFilter({
+                ...searchFilter,
+                Cluster: searchFilter.Cluster || '*',
+                standard: standardQueryValue,
+            });
+            const link = `${complianceBasePath}/controls?${query}`;
```

Comment:
any reason why you opted to use `controls` vs ``urlEntityListTypes[standardEntityTypes.CONTROL]` as it was before?

Reply:
Good eyes. Similar reason as consensus in team meeting for constants for sort fields to be container scope instead of global scope when possible. Especially because `controls` is unique to compliance.